### ISSUE TITLE
No CSS on https://ocaml-multicore.github.io/multicoretests/dev/

### ISSUE THREAD

**Issue by jmid** [ocaml-multicore/multicoretests#335]
Currently we get a 404 on CSS related to https://ocaml-multicore.github.io/multicoretests/dev/

I believe a recent change to `odoc` caused it to generate an `_odoc_support` directory with JS and CSS.
However the choice of directory name (beginning with an underscore) causing Jekyll, the site generator underlying Github pages, to ignore the directory:
```
$ curl -I https://ocaml-multicore.github.io/multicoretests/dev/_odoc_support/odoc.css
HTTP/2 404 
server: GitHub.com
...
```

This is a known issue
https://medium.com/@juliakmdev/learning-that-github-pages-struggles-with-certain-file-names-4d867d499ca9
so I've tried a few different recommendations like adding a `.nojekyll` and adjusting `_config.yml`, but I'm clearly missing something! :grimacing: 

The issue is known in the OCaml community too, with a WIP PR in dune to output to a less problematic directory name, which should solve the issue for us...

**Comment by shym** [comment]
I see you’ve tried many things already but maybe the one suggested here, aka `include: [_odoc_support]` would work?
https://talk.jekyllrb.com/t/include-all-files-in-a-specific-directory-even-if-the-filenames-start-with-underscores/5849/8

**Comment by jmid** [comment]
Bingo - thanks a ton! :100: :pray: 
Now fixed in 8843e98 :smiley:
